libraries/raw-rs/src/postprocessing/convert_to_rgb.rs

```diff
@@ -0,0 +1,36 @@
+use crate::Image;
+
+pub fn convert_to_rgb(mut image: Image<u16>) -> Image<u16> {
+	let Some(rgb_to_camera) = image.rgb_to_camera else { return image };
+
+	let channels = image.channels as usize;
```

Comment:
`channels` always refers to the number of channels that is stored in the `Image`. This could be 3 or 4. The bayer filter uses only 3 channels but RGBE filter and CYGM filter uses 4. So after demosaicing, there can be 4 channels. And this step is also supposed convert such formats back to rgb using the `rgb_to_camera`. Hence it is hard-coded to 3 in this step. In fact, for cameras with 4 channel bayer filter the matrix will be of size 3x4 instead of 3x3. All of these changes have not been made till now since there is no camera in the 40 camera models with 4 channel bayer filter so I decided to postpone it to later. If such changes should be added is another story since there is only one camera with RGBE filter released in 2003 (Sony Cyber-shot DSC-F828) and the most recent camera using CYGM was released in 2002. So I doubt anybody uses it now but I could be wrong. 

Should I completely assume that such filter don't exist? This will simplify the code too.

Reply:
Let's keep it as-is but I've added some small bits of documentation to clarify this.